src/lib.rs

```diff
@@ -43,6 +43,13 @@ pub use towkt::ToWkt;
 #[cfg(feature = "geo-types")]
 pub mod conversion;
 
+#[cfg(feature = "serde")]
+extern crate serde;
+#[cfg(feature = "serde")]
+pub mod deserialize;
+#[cfg(feature = "serde")]
```

Comment:
ah! it's failing because we need both serde and geo-types for this method.

Reply:
Ah right, I failed this one. Nice CI setup to catch that by the way. Fixed in 2c6f669686dc334ada95442672a7a52418e85df9.